### PR TITLE
Don't choke on messages containing newlines

### LINKED ISSUE #19: weirdness when message has a newline character
I know this is kind of an edge case, but I noticed when I accidentally used a message string that had a newline character in it. `pb` doesn't deal with this case (either by stripping newlines or by drawing the progress bar differently), so it just spews out progress bar lines.

### PR DESCRIPTION
**Pull request by nabijaczleweli**
Just replace them with spaces

Closes #19

### CODE REVIEW

File: src/pb.rs [hunk 1]
```diff
@@ -196,7 +198,7 @@ impl<T: Write> ProgressBar<T> {
     ///
     /// ```
     pub fn message(&mut self, message: &str) {
-        self.message = message.to_owned()
+        self.message = message.to_owned().replace("\n", " ").replace("\n", " ")
```
Comment (a8m) [inline src/pb.rs]: did you mean to replace `\r` at the second call?

Comment (nabijaczleweli) [inline src/pb.rs]: Yes :v
Copy-paste programming at its finest

Comment (nabijaczleweli) [inline src/pb.rs]: Fixed

**Comment by a8m** [comment]
Awesome @nabijaczleweli, thanks

**Pull request merged**